staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -144,6 +177,9 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.OpenAPIV3Config.Info.Title = "Wardle"
 	serverConfig.OpenAPIV3Config.Info.Version = "0.1"
 
+	serverConfig.FeatureGate = utilversion.DefaultComponentGlobalsRegistry.FeatureGateFor(utilversion.DefaultKubeComponent)
```

Comment:
why isn't this using the wardle component?

Reply:
updated.